public/js/bio-link/settings.js

```
@@ -0,0 +1,864 @@
+            document.getElementById('bioYoutube').value = data.social.youtube;
+        }
+        if (data.social.tiktok) {
+            document.getElementById('bioTiktok').value = data.social.tiktok;
+        }
+        if (data.social.github) {
+            document.getElementById('bioGithub').value = data.social.github;
+        }
+    }
+
+    if (data.links && data.links.length > 0) {
+        document.getElementById('bioLinksListContainer').innerHTML = '';
```

ESLint

[error] 'document' is not defined.

(no-undef)